src/Database.hpp

```diff
@@ -63,6 +63,8 @@ namespace Database {
         int angle;
         short int PCState;
         std::vector<char> QuestFlag;
+        int64_t iFirstUseFlag1;
+        int64_t iFirstUseFlag2;
```

Comment:
```suggestion
        uint32_t iFirstUseFlag2;
```
32-bit and unsigned. You could have also used an array of length 2, but it doesn't matter.

Reply:
I didn't use an array to imitate client code and get it over with.
As for int64_t, this is how it was already defined in `0104.hpp` so I copied them.